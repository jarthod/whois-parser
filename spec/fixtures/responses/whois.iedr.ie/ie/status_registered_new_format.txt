Domain Name: google.ie
Registry Domain ID: 762999-IEDR
Registrar WHOIS Server: whois.weare.ie
Registrar URL: http://www.eMarkmonitor.com
Updated Date: 2025-05-05T14:18:36Z
Creation Date: 2002-03-21T00:00:00Z
Registry Expiry Date: 2026-03-21T14:13:27Z
Registrar: Markmonitor Inc
Registrar IANA ID: not applicable
Registrar Abuse Contact Email: abusecomplaints@markmonitor.com
Registrar Abuse Contact Phone: +1.2086851865
Domain Status: serverDeleteProhibited https://icann.org/epp#serverDeleteProhibited
Domain Status: serverTransferProhibited https://icann.org/epp#serverTransferProhibited
Domain Status: serverUpdateProhibited https://icann.org/epp#serverUpdateProhibited
Registry Registrant ID: 379250-IEDR
Registrant Name: Google LLC
Registry Admin ID: 53735502-IEDR
Registry Tech ID: 534389-IEDR
Registry Billing ID: REDACTED FOR PRIVACY
Name Server: ns1.google.com
Name Server: ns2.google.com
Name Server: ns3.google.com
DNSSEC: unsigned
>>> Last update of WHOIS database: 2025-08-02T15:32:41Z <<<

% Important Notice
% If you believe that information published on this page is incorrect, or should not be published,please contact your Registrar, or the .IE Registration Services Department who will advise you further.
% You can also contact your Registrar or .IE if your Registrant name is not showing and you would like it to be published.
% Our contact information is available at https://www.weare.ie/contact-us/
% Rights restricted by copyright: https://www.weare.ie/registrant-terms-conditions-policy/
% Do not remove this notice
