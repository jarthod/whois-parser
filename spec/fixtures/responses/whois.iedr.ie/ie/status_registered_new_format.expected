#disclaimer
  %s == "Important Notice If you believe that information published on this page is incorrect, or should not be published,please contact your Registrar, or the .IE Registration Services Department who will advise you further. You can also contact your Registrar or .IE if your Registrant name is not showing and you would like it to be published. Our contact information is available at https://www.weare.ie/contact-us/ Rights restricted by copyright: https://www.weare.ie/registrant-terms-conditions-policy/ Do not remove this notice"

#domain
  %s == "google.ie"

#domain_id
  %s == "762999-IEDR"

#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true

#created_on
  %s %CLASS{time}
  %s == Time.parse("2002-03-21T00:00:00Z")

#updated_on
  %s %CLASS{time}
  %s == Time.parse("2025-05-05T14:18:36Z")

#expires_on
  %s %CLASS{time}
  %s == Time.parse("2026-03-21T14:13:27Z")

#registrar
  %s %CLASS{registrar}
  %s.name == "Markmonitor Inc"

#registrant_contacts
  %s %CLASS{array}
  %s.size == 1
  %s[0] %CLASS{contact}
  %s[0].type == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].name == "Google LLC"
  %s[0].id == "379250-IEDR"

#admin_contacts
  %s %CLASS{array}
  %s.size == 0

#technical_contacts
  %s %CLASS{array}
  %s.size == 0

#nameservers
  %s %CLASS{array}
  %s.size == 3
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.google.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.google.com"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns3.google.com"
